src/Views/Form-Themes/Sequoia/config.php

```diff
@@ -0,0 +1,127 @@
+<?php
```

Comment:
@henryholtgeerts This file has been removed in the latest `feature/form-theme`. Please check latest branch.

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/50b98c62bc9bfdb94a7d540f785af3675a59fb0e